cloud/openstack/os_floating_ip.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# Copyright (c) 2014 Hewlett-Packard Development Company, L.P.
+# Copyright (c) 2013, Benno Joy <benno@ansible.com>
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+
+try:
+    import shade
+    HAS_SHADE = True
+except ImportError:
+    HAS_SHADE = False
+
+DOCUMENTATION = '''
+---
+module: os_floating_ip
```

Comment:
Hate to bring it up now, but looks like the python-openstackclient folks decided that the verbiage should be "ip floating", such as "ip floating create" and "ip floating add". Should our modules follow suit here, to try and reduce the variety of ways in which these resources are referenced?